### ISSUE TITLE
Does the project still have support?

### ISSUE THREAD

**Issue by zaqueoae** [phoboslab/JavaScriptCore-iOS#36]
Hi,

I am developing a project for ios. I wanted to ask if the project is still alive.

Thank you.

**Comment by phoboslab** [comment]
No, it's not. Why would you still want to use it? You can now use the JavaScriptCore library provided by the OS.

**Comment by vovkasm** [comment]
> Why would you still want to use it? You can now use the JavaScriptCore library provided by the OS.

Because different versions of iOS have different versions of JavaScriptCore framework with different features. For example mobx 5 (requires Proxy) will not work in JSC from iOS9.
